app/src/main/java/com/futurice/freesound/feature/search/SearchFragment.java

```diff
@@ -150,7 +150,7 @@ private void showNothing() {
         get(resultsRecyclerView).setVisibility(View.GONE);
     }
 
-    private void showResults(@NonNull final List<Sound> sounds) {
+    private void showResults(@NonNull final List<DisplayableItem> sounds) {
```

Comment:
The `get` call should occur earlier; currently if `sounds` was `null` it would throw on the first `if` statement.